cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -2,18 +2,14 @@ name "Azure Hybrid Use Benefit for Windows Server"
 rs_pt_ver 20180301
 type "policy"
 short_description "Identifies eligible instances not utilizing Azure Hybrid Use Benefit.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/hybrid_use_benefit) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
I suggest changing this to **Identifies instances eligible for Azure Hybrid Use Benefit.**

Reply:
Updated this